crates/rolldown/tests/rolldown/function/comments/preserve/artifacts.snap

```diff
@@ -0,0 +1,16 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Assets
+
+## main.js
+
+```js
```

Comment:
I'm not sure why it doesn't work. I check the passing value is indeed `comments: true`, so let's not block merging this PR due to it doesn't work.

@Boshen cc

Reply:
Does `print_all_comments` should be superset of `print_legal_comments_only` ?

It's wired that similar case is passed for `print_legal_comments_only`.

https://github.com/rolldown/rolldown/pull/2615/files#diff-dc8ff37288ec95c0339a59b4e355c861f7ad41ad1f24af5025a5548d369fbc93